src/redist/targets/BundledManifests.targets

```diff
@@ -6,7 +6,8 @@
     <BundledManifests Include="Microsoft.NET.Sdk.MacCatalyst.Manifest-6.0.100" Version="$(XamarinMacCatalystWorkloadManifestVersion)" WorkloadManifestId="Microsoft.NET.Sdk.MacCatalyst" />
     <BundledManifests Include="Microsoft.NET.Sdk.macOS.Manifest-6.0.100" Version="$(XamarinMacOSWorkloadManifestVersion)" WorkloadManifestId="Microsoft.NET.Sdk.macOS" />
     <BundledManifests Include="Microsoft.NET.Sdk.tvOS.Manifest-6.0.100" Version="$(XamarinTvOSWorkloadManifestVersion)" WorkloadManifestId="Microsoft.NET.Sdk.tvOS" />
-    <BundledManifests Include="Microsoft.NET.Sdk.BlazorWebAssembly.AOT" Version="$(BlazorWorkloadManifestVersion)" WorkloadManifestId="Microsoft.NET.Workload.BlazorWebAssembly" />
+    <BundledManifests Include="Microsoft.NET.Sdk.tvOS.Manifest-6.0.100" Version="$(XamarinTvOSWorkloadManifestVersion)" WorkloadManifestId="Microsoft.NET.Sdk.tvOS" />
+    <BundledManifests Include="Microsoft.NET.Workload.Mono.ToolChain.Manifest-6.0.100" Version="$(MonoWorkloadManifestVersion)" WorkloadManifestId="Microsoft.NET.Workload.Mono.ToolChain.Manifest" />
```

Comment:
FYI @danroth27. This would change the workload name you would use in `dotnet workload install`

Reply:
Also FYI @joeloff since this would now replace the Blazor workload.

Reply:
Yeah, this doesn't look very discoverable or intuitive. @lewing can we discuss alternatives?

Reply:
Which name are you concerned about? The current package the Manifest package name is not really a user facing feature.  Or are you worried about the workload-id in `dotnet workload install <workload-id>`?

Reply:
If it is the workload-id I meant to have a longer discussion about the ids but I've reverted that change so once the packages update the command will still be `dotnet workload install microsoft-net-sdk-blazorwebassembly-aot`

Reply:
Yeah, I was asking about the workload ID that users would need to use when doing `dotnet workload install`.

Reply:
Yeah I think should still discuss the id string but mostly about making it shorter not making the blazorwasm-aot connection clear (it's simple enough to have others for other targets)

Reply:
Why does the mono manifest include "manifest" in the manifest ID, while the other don't?

Reply:
> Why does the mono manifest include "manifest" in the manifest ID, while the other don't? 

The installer spec seems to call for it.

https://github.com/dotnet/designs/pull/188/commits/fddce9904794e3515136a61d73ae26f387f7db64

Reply:
> Why does the mono manifest include "manifest" in the manifest ID, while the other don't?

basically we changed from using Sdk to Workload (after asking) at the almost exactly same time the xamarin workloads changed from Workload to Sdk in https://github.com/dotnet/installer/pull/10538